main/acle.md

```diff
@@ -7156,6 +7158,26 @@ Assuming `float64_t` data, the C version of the code above would be:
   p3 = svrdffr();
 ```
 
+### SVE reinterpret intrinsics
+
+The `svreinterpret` intrinsics for vector types and vector tuple types take the form
+
+``` c
+svreinterpret_type0[_type1][_xN]
```

Comment:
According to the example bellow this should be
svreinterpret_type0[_type1_xN],
because you don't have : uin16x2_t svreinterpret_u16_x2